tardis/transport/montecarlo/tests/conftest.py

```diff
@@ -27,6 +28,15 @@ def nb_simulation_verysimple(config_verysimple, atomic_dataset):
     sim.iterate(10)
     return sim
 
+@pytest.fixture(scope="package")
+def simple_weighted_packet_source():
+    packet_source = BlackBodyWeightedSource(
```

Comment:
I don't think we need legacy mode to test a new source unless you are comparing it to existing unweighted source output?

Reply:
Nope, I'll remove it